cpp/test/sparse/distance.cu

```diff
@@ -105,21 +79,20 @@ class SparseDistanceTest
     dist_config.a_indptr = indptr;
     dist_config.a_indices = indices;
     dist_config.a_data = data;
-    dist_config.handle = cusparseHandle;
-    dist_config.allocator = alloc;
-    dist_config.stream = stream;
 
     int out_size = dist_config.a_nrows * dist_config.b_nrows;
 
     allocate(out_dists, out_size);
 
     pairwiseDistance(out_dists, dist_config, params.metric, params.metric_arg);
 
-    CUDA_CHECK(cudaStreamSynchronize(stream));
+    CUDA_CHECK(cudaStreamSynchronize(handle.get_stream()));
+
+    raft::print_device_vector("dists", out_dists, out_size, std::cout);
```

Comment:
We should either remove this print or use a conditional to print only when the test fails.

Reply:
I think removing seems better, I'll do it